x-pack/plugins/observability_solution/slo/server/services/get_slo_suggestions.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { SavedObjectsClientContract } from '@kbn/core/server';
+import { SO_SLO_TYPE } from '../saved_objects';
+type Buckets = Array<{
+  key: string;
+  doc_count: number;
+}>;
+
+interface AggsResponse {
+  tagsAggs: {
+    buckets: Buckets;
+  };
+}
```

Comment:
Do we want to introduce a type in slo-schema, so we can also import it in the hook/ui?